src/lib/PriorityOrderLib.sol

```diff
@@ -5,6 +5,11 @@ import {OrderInfo, InputToken, OutputToken} from "../base/ReactorStructs.sol";
 import {OrderInfoLib} from "./OrderInfoLib.sol";
 import {ERC20} from "solmate/src/tokens/ERC20.sol";
 
+struct PriorityCosignerData {
+    // the block at which the order can be executed (overrides auctionStartBlock)
+    uint256 auctionTargetBlock;
```

Comment:
can we flatten this, or do you think it's better to keep the same structure as v2 Order

Reply:
I think better to keep as same structure since not much overhead for user as they are not signing over this